ci/gitlab/bin/quicc/gitlab/pipelines.py

```diff
@@ -68,6 +69,9 @@ def base_yaml(self):
                         {
                             'DOCKERFILE': self.base_docker,
                             'PERSIST_IMAGE_NAME': self.base_path_image,
+                            'SECONDARY_IMAGE_NAME': self.dockerhub_base_image,
```

Comment:
Is this just to provide an alternative location and make it CSCS agnostic?

Reply:
it is to push the docker images also to dockerhub, so the users can access them easily